src/show/pages/public/CreateProject/index.js

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+import { useState } from 'react';
+import StepWizard from 'react-step-wizard';
+import { useNavigate } from 'react-router-dom';
+import { PROJECT_PATH } from '../../../../process/routes/paths';
+
+import {
+  FeeStep,
+  GoalStep,
+  TitleStep,
+  ImagesStep,
+  PublishStep,
+  CategoryStep,
+  DatelineStep,
+  StatementStep,
+  DescriptionStep,
+  TeamMembersStep
+} from '../CreateProject/Subviews';
+
+const CreateProject = () => {
+  const navigate = useNavigate();
+  const [project, setProject] = useState({
```

Comment:
Empty line before this and make an initial state for every field that you are adding/editing (just like you did for images, team_members)

Reply:
done